src/components/WwwFrame/WwwPage.vue

```diff
@@ -67,14 +82,27 @@ export default {
 		preFetch(config, client, { route }) {
 			return Promise.all([
 				client.query({ query: hasEverLoggedInQuery }),
+				client.query({ query: optimizelyUserDataQuery }),
 				fetchAllExpSettings(config, client, {
 					query: route?.query,
 					path: route?.path
 				})
 			]);
-		},
+		}
 	},
 	created() {
+		if (this.cookieStore.get(hasLentBeforeCookie) === undefined || this.cookieStore.get(hasDepositBeforeCookie) === undefined) { // eslint-disable-line max-len
```

Comment:
This check would also need to be a conditional for including the new `client.query({ query: optimizelyUserDataQuery })` in the preFetch. [Here's an example of a conditional prefetch](https://github.com/kiva/ui/blob/main/src/pages/Thanks/ThanksPage.vue#L189)

Reply:
done!